### DISCUSSION TITLE
The service "security.command.debug_firewall" has a dependency on a non-existent service

### DISCUSSION THREAD

**Discussion by Ralfefernandes** [pimcore/pimcore#13677] [Q&A / Support]
Hi Pimcore,

I Just upgrade Pimcore Enterprise from 6.9 to version 10.0

Now I’m facing this problem:
`The service "security.command.debug_firewall" has a dependency on a non-existent service "Pimcore\Bundle\AdminBundle\Security\Authenticator\AdminLoginAuthenticator`

By running any of this `command: bin/console, bin/console c:c`

What I did until now to resolve this issues:
I checked twice the` security.yaml` inside of `config/packages/security.yaml` and i compare the file with Pimcore-skeleton and there is no difference.
Is there someone with the same issues ?
<img width="960" alt="2022-11-24 (3)" src="https://user-images.githubusercontent.com/12101478/203791339-64680e35-da53-4795-adde-df0f9d24bedb.png">

**Comment by Ralfefernandes** [comment]
Farther investigation, I see that by upgrading to Pimcore 10, Pimcore didn’t install those Pimcore Bundles:
```
- Pimcore\Bundle\AdminBundle\Security\Authentication\AdminLoginAuthenticator
- Pimcore\Bundle\AdminBundle\Security\Authenticator\AdminTokenAuthenticator
- Pimcore\Bundle\AdminBundle\Security\Authenticator\AdminSessionAuthenticator
```
How to get them?

**Reply by dvesh3** [reply]
Since you are trying to use [New Security Authenticator](https://pimcore.com/docs/pimcore/current/Development_Documentation/Development_Tools_and_Details/Security_Authentication/Authenticator_Based_Security.html), please make sure that you are using Pimcore version >= 10.5 since authenticator was introduced in this version.

**Comment by Ralfefernandes** [comment]
By upgrading to version 10.5. I'm getting now this error:
<img width="960" alt="2022-11-24 (7)" src="https://user-images.githubusercontent.com/12101478/203830639-a4e79858-c048-4fc0-8e1a-aef99f6538eb.png">

`In AbstractMySQLDriver.php line 128:`
`An exception occurred in driver: SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost'`
`SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost'`

Do you know how i can fix it?

**Reply by Ralfefernandes** [reply]
> By upgrading to version 10.5. I'm getting now this error: <img alt="2022-11-24 (7)" width="960" src="https://user-images.githubusercontent.com/12101478/203830639-a4e79858-c048-4fc0-8e1a-aef99f6538eb.png">
> 
> `In AbstractMySQLDriver.php line 128:` `An exception occurred in driver: SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost'` `SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost'`
> 
> I just remove the symfony/symfony packages and everything is now working properly
